app/models/configurable.rb

```diff
@@ -29,6 +29,14 @@ def config_field(key, options = {})
       end
     end
 
+    def merge_configuration_fields
+      if self.superclass.respond_to? :merge_configuration_fields
```

Comment:
that is certainly neater, i'll give that a try!

Reply:
unfortunately no, that does not work. we need to know specifically if the parent class has mixed in `Configurable` so that we can collect its configuration fields as well, so it's not just a question of whether the class has a parent or inherits from a parent, but whether that method exists.